srcpkgs/cross-powerpc-linux-musl/template

```diff
@@ -0,0 +1,223 @@
+# Template build file for 'cross-powerpc-linux-musl'
+#
+_binutils_version=2.31.1
+_gcc_version=8.2.0
+_musl_version=1.1.20
+_linux_version=4.19
+_triplet=powerpc-linux-musl
+_fpuflags=
+_archflags="-mcpu=powerpc -msecure-plt"
+_sysroot="/usr/${_triplet}"
+pkgname=cross-${_triplet}
+version=0.27
+revision=1
+short_desc="Cross toolchain for PowerPC (musl)"
+maintainer="newbluemoon <blaumolch@mailbox.org>"
```

Comment:
I think you should be here as maintainer.

Reply:
unless @newbluemoon is ok with being the maintainer of this

Reply:
Well, I initially created this package and don’t mind maintaining it. But I would also be ok if someone else would like to take maintainership. :)